panel/widgets/select.py

```diff
@@ -654,6 +673,73 @@ def _update_options_programmatically(self):
                 self.value = original_values
                 raise
 
+    @classmethod
+    def create_options_from_dataframe(cls, data: pd.DataFrame, columns: List|None=None)->Dict|List:
+        """Returns the options to use with a NestedSelect
+
+        Args:
```

Comment:
These docstrings do not follow the style we use throughout Panel, i.e. they're Google style not NumPy style.